documentation/src/test/java/org/hibernate/search/documentation/search/predicate/PredicateDslIT.java

```diff
@@ -1106,7 +1113,8 @@ void not() {
 
 	@Test
 	void knn() {
-		// TODO HSEARCH-4950 Remove this assumption when we support KNN search using Elasticsearch/OpenSearch
+		// NOTE: To keep this documentation example simple there is no testing with Elasticsearch/OpenSearch
+		// as not all versions have integration implemented e.g. Elasticsearch 7 or OpenSearch 1.3 will throw exceptions
```

Comment:
I think you can call `org.hibernate.search.util.impl.integrationtest.backend.elasticsearch.dialect.ElasticsearchTestDialect#isActualVersion(...)` from here, if it helps?